modules/migration-adding-replication-repository-to-cam.adoc

```diff
@@ -4,14 +4,19 @@
 // migration/migrating-4-4/migrating-openshift-4_1-to-4.adoc
 // migration/migrating-4-4/migrating-openshift-4_2-to-4.adoc
 [id='migration-adding-replication-repository-to-cam_{context}']
-= Adding a Repository to the CAM web console
+= Adding a replication repository to the CAM web console
 
-You can add an S3 repository to the CAM web console.
+You can add a replication repository to the CAM web console.
+
+.Prerequisites
+
+* The storage provider must support the S3 API.
+* You must configure an S3 bucket for migrating the data.
```

Comment:
Change to:

You must configure an Object Storage bucket for migrating the data.